### DISCUSSION TITLE
View Predefined Module Parameters?

### DISCUSSION THREAD

**Discussion by mrwildbob** [arendst/Tasmota#14324] [Q&A] [answered]
How do you view the predefined Module Parameters?  It would be a nice feature to be able to at lease see what the settings are for the different modules.

boB

**Comment by sfromis** [comment, marked as answer]
If you go into "Configure Template", and select the predefined module as "Based on", then you'll see the dropdowns for each gpio populated with the pins for that module. You do not need to hit save, but just exit again. Of course, you can also create a modified version to save as a template, if that's relevant to you.
